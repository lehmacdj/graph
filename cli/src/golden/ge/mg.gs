mk a + b
mg a + b
cd a
d
ls
