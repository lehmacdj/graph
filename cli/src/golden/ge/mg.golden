==========stdout==========
g> g> g> g> g> "a" at 0
"b" at 0
g> Goodbye!
==========stderr==========
==========graph-dump==========
0{in=[], out=[1 via "a",1 via "b"]}
1{in=[0 via "a",0 via "b"], out=[]}
